app/page.tsx

```diff
@@ -1,31 +1,30 @@
-import Image from "next/image";
+"use client";
+
+import { Login } from "@/components/organisms/Login/Login";
+import { Chat } from "@/components/organisms/Chat/Chat";
+import { useState } from "react";
 
 export default function Home() {
+  const [salesforceId, setSalesforceId] = useState("");
```

Comment:
Why default string?

Reply:
Should i make it undefined or something? didnt really see a difference tho